packages/account/src/Containers/account.jsx

```diff
@@ -174,25 +168,6 @@ const Account = ({
         history.push(shared_routes.personal_details);
     }
 
-    const action_bar_items = [
```

Comment:
deleting this as it isn’t used

Reply:
@yauheni-deriv why do you think we don't use it? as long as we have routes passed into the component, there can be use cases, no?

Reply:
Because here in account.jsx we don’t pass this variable anywhere.

Reply:
@yauheni-deriv we are using it at line 190 which you also deleted for some reason👀